source/custom/xnetsocket_converters.h

```diff
@@ -24,27 +24,22 @@ using ResourceRepositorySharedPtr_ = std::shared_ptr<nidevice_grpc::SessionResou
 struct SockAddrInputConverter {
   SockAddrInputConverter(const SockAddr& input)
   {
-    // memset zero ensures:
-    // (a) any unset bytes in the ipv6 addr are zero.
-    // (b) If neither oneof case is selected, sa_familly will be nxAF_UNSPEC.
-    std::memset(&addr, 0, sizeof(addr));
```

Comment:
Thanks to @reckenro for the tip to use braced initialization for these instead of memset. I've updated that throughout. With matching `{}` in the member declarations.